layers/+lang/ipython-notebook/packages.el

```diff
@@ -157,4 +158,11 @@
         ("9" ein:notebook-worksheet-open-last)
         ("+" ein:notebook-worksheet-insert-next)
         ("-" ein:notebook-worksheet-delete)
-        ("x" ein:notebook-close)))))
+        ("x" ein:notebook-close))
+      (spacemacs/set-leader-keys "ein" 'spacemacs/ipython-notebook-transient-state/body))))
+
+(defun ipython-notebook/init-ob-ipython ()
+  (use-package ob-ipython
+    :defer t
```

Comment:
If this is deferred here what causes it to be loaded? I tested it out and it seems like ob-ipython is not loaded when visiting a `.org` buffer, but has to be explicitly `require`d. Should we remove `:defer t` from this config?